src/GameScore/GameScore.cpp

```diff
@@ -7,258 +6,231 @@
 
 GameScore::GameScore() {}
```

Comment:
Could we define this or remove it entirely? If there's nothing to specifically inititalize I think we can just the one provided by the compiler.